exercice 7 Standardization.py

```diff
@@ -0,0 +1,12 @@
+import numpy as np
+def standardization(A) :
```

Comment:
very interesting function !

Reply:
A better one is possible thanks to a better use of numpy !